android/app/src/main/java/com/microsoft/codepush/react/CodePushConstants.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.codepush.react;
+
+public class CodePushConstants {
+    static final String ASSETS_BUNDLE_PREFIX = "assets://";
```

Comment:
Leaving out the access modifier makes it "package private", which I thought more closely resembles the behaviour in the Windows implementation.